IO.Eventuate.Tram/Consumer/Kafka/QueuedMessage.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using IO.Eventuate.Tram.Messaging.Common;
+
+namespace IO.Eventuate.Tram.Consumer.Kafka;
+
+internal class QueuedMessage
```

Comment:
Consider adding a description to explain why this wrapper exists.

Reply:
ok, I'll write something